src/vue/mixins/verification-form.mixin.js

```diff
@@ -34,9 +34,11 @@ export default {
     },
     createKycOperation (kycBlobId, accountRole) {
       return base.CreateChangeRoleRequestBuilder.createChangeRoleRequest({
-        requestID: this.kycState === REQUEST_STATES_STR.rejected
-          ? this.kycRequestId
-          : KYC_CREATION_REQUEST_ID,
+        requestID:
+          this.kycState === REQUEST_STATES_STR.rejected ||
+          this.kycState === REQUEST_STATES_STR.pending
```

Comment:
correct me if I’m wrong, but I thought pending requests are not updatable yet? The task says we should allow updating of approved requests, not pending ones

Reply:
> Pending KYC requests cannot be updated, but approved ones — can. Please add ability of updating KYC requests

Just read it likes: "approved" can be updated, but "pending" – not. Need be allowed to update "pending".
Pending requests will updatable with this PR on the `Client`-side. Back-end allows updating "pending" as long as I live. Remove this feature and add updating "approved" instead or leave it and add the ability to update "approved" too?

Reply:
Just discussed it with back-end guys, they say that some pending requests *can* be updated, but earlier some of the said they *cannot* be updated, sorry for misleading. The indicator of updatable-pending-request is presence of `requestId` field in the builder.

Thus, we have no state that cannot be updated. So we truly can remove this `isEditable` check and act in all the other places accordingly. The only point we should not ignore is that when you try updating a request that was _rejected permanently_ we should not try updating the old request but creating the new one (since the old request is deleted if it was rejected permanently).